docs/function-mesh-worker/function-mesh-worker-overview.md

```diff
@@ -16,6 +16,34 @@ The following figure illustrates how Function Mesh Worker service works with Pul
 
 ![Function Mesh Workflow](./../assets/function-mesh-workflow.png)
 
+## Authentication Support
+
+Before v0.6.0, the Function Mesh Worker Service uses a super admin Pulsar account reading from the config file for Function Mesh operators, which is not secure, so start from v0.6.0, there is a break change to use separate account for each user.
+
+**Currently it only support OAuth2 providers, other providers will be supported as soon as possible.**
+
+> NOTE: if you are using Pulsar cluster with other authentication providers, the Function Mesh Worker Service will not work
```

Comment:
we can remove this note as we have expressed similar meaning in the previous sentence.